src/_base/harness/scripts/enable.sh.twig

```diff
@@ -85,7 +85,8 @@ dynamic_console()
 
     find .my127ws/docker/image/ -type d ! -perm u+rwx,go+rx,o-w -exec chmod u+rwx,go+rx,o-w {} +
 
-    passthru docker-compose up --build -d console
+    passthru docker-compose up --build -d console --wait
+    # docker-compose will wait for `console` to be in healty status if healthcheck is enabled
```

Comment:
comments should be before a line they are referring to, or otherwise in-line (but former is best)
```suggestion
    # docker-compose will wait for `console` to be in healty status if healthcheck is enabled
    passthru docker-compose up --build -d console --wait
```